### ISSUE TITLE
Unable to save changes if Magnifier panel has been loaded in in NVDA 2026.2 beta 2

### ISSUE THREAD

**Issue by Bibeksharma5** [nvaccess/nvda#20337]
### Brief summary

Hello NVAccess team, I'm facing an issue in NVDA 2026.2 beta 2 that the changes can't be saved in math settings. When I click OK button or press enter in the settings interface, an error sound plays and nothing happens. If I click cancel or press escape to exit settings, next time, if I go to NVDA menu>Preferences>Settings, the NVDA settings is not opening. That's why, please fix this bug.

### Steps to reproduce

1. Open NVDA settings and go to math category.
2. Do any changes and press enter or click OK button.
3. Nothing happened. Press escape to go back and again open the NVDA settings. The settings can't be opened until the NVDA restarts.

### Actual behavior

Explained above
Speech viewer:
"NVDA Speech Viewer
Show Speech Viewer on Startup  check box  not checked  Alt+  s
NVDA+n
NVDA  menu
down arrow
Preferences  subMenu  p
right arrow
Settings...  s
enter
NVDA Settings: General (normal configuration)  dialog
Categories:  list  Alt+  c
General  1 of 20
Mouse  8 of 20
Magnifier  15 of 20
Math  17 of 20
tab
Math  property page  The following options control the presentation of mathematical content.
Speech  grouping
Impairment:  combo box  Blindness  collapsed
tab
Language:  combo box  English  collapsed
tab
Decimal separator for numbers:  combo box  Automatic  collapsed
tab
Speech style:  combo box  ClearSpeak  collapsed
tab
Speech verbosity:  combo box  Medium  collapsed
tab
Relative speech rate  slider  100
tab
Pause factor  slider  50
tab
Beep at the beginning and end of math  check box  not checked
tab
Chemical formulae:  combo box  Spell it out (H 2 O)  collapsed
shift+tab
Beep at the beginning and end of math  check box  not checked
checked
enter
not checked
enter
shift+tab
Pause factor  slider  50
left arrow
42
left arrow
35
left arrow
28
left arrow
21
enter
tab
Beep at the beginning and end of math  check box  not checked
tab
Chemical formulae:  combo box  Spell it out (H 2 O)  collapsed
tab
When entering an equation:  combo box  Speak  collapsed
tab
Navigation  grouping
Default navigation mode:  combo box  Enhanced  collapsed
tab
Reset to the default navigation mode for each new equation  check box  checked
tab
Automatically zoom out of two-dimensional notation  check box  checked
tab
Navigation verbosity:  combo box  Medium  collapsed
tab
Copy math as:  combo box  MathML  collapsed
tab
Braille  grouping
Output code:  combo box  Automatic (UEB)  collapsed
tab
Highlight navigation focus with dots 7 and 8  combo box  Endpoints  collapsed
tab
Application support  grouping
Use native math support in Word and Outlook  check box  not checked
tab
OK  button
pressed
enter
tab
Cancel  button
pressed
NVDA Speech Viewer
Show Speech Viewer on Startup  check box  not checked  Alt+  s
NVDA+n
NVDA  menu
down arrow
Preferences  subMenu  p
right arrow
Settings...  s
enter
NVDA Speech Viewer
Show Speech Viewer on Startup  check box  not checked  Alt+  s
tab"

### Expected behavior

There should be no error sound and the settings should be saved correctly.

### NVDA logs, crash dumps and other attachments

[nvda.log](https://github.com/user-attachments/files/28924740/nvda.log)
[nvda_crash.dmp](https://github.com/user-attachments/files/28924741/nvda_crash.dmp)
[nvda_crash_stats.txt](https://github.com/user-attachments/files/28924742/nvda_crash_stats.txt)
[nvda-old.log](https://github.com/user-attachments/files/28924743/nvda-old.log)

### NVDA type

installed copy

### NVDA version

2026.2beta2 (2026.2.0.56391)

### Have you tried any other versions of NVDA? If so, please report their behaviors.

I don't know whether this issue was in 2026.2 beta 1 or not. But, in 2026.1, this issue doesn't exist.

### Windows version

Windows 11 25H2 (AMD64) build 26200.8655

### Does the issue still occur after restarting your computer?

I have not restarted my computer

### If NVDA add-ons are disabled, is your problem still occurring?

I have restarted NVDA with add-ons disabled and the issue still occurs

### Does the issue still occur after you run the System Accessibility Repair Tool (COM Registration Fixing Tool in older versions) in NVDA's tools menu?

I have not run the System Accessibility Repair Tool

**Comment by Bibeksharma5** [comment]
Hi! its me again Bibek. I want to make a small correction that this issue still occurs after restarting the system. Thanks.

**Comment by SaschaCowley** [comment]
Hi, the issue here is actually not with the Math settings panel, but with Magnifier's settings panel, which is loaded if arrowing down to the Math category, or navigating there with 'M'.

@Bibeksharma5 in the meantime, you should be able to open the settings dialog, press `end` to jump to the last settings category, then use `upArrow` to get to Math, change the settings you want to change, and press OK.

CC @Boumtchack, @CyrilleB79

**Comment by CyrilleB79** [comment]
The issue has already been fixed in #20261. Sorry, it has not been clearly advertised in the PR's description. Thanks.
